### ISSUE TITLE
Components don't always render if there are not trailing spaces in the tag

### ISSUE THREAD

**Issue by trvrm** [ractivejs/ractive#2765]
## Description:
Under some circumstances, when referring to a component in a template, it silently fails to render if there is no trailing space before the tag close.

The documentation at http://docs.ractivejs.org/edge/components indicates that 

`<widget message='{{foo}}'/>` 

should work, but I find that in many cases I must include whitespace before the `/>` for the component to render.

## Versions affected:

0.8.5

## Platforms affected:

Chrome

## Reproduction:
http://codepen.io/trvrm/pen/PbGYxp
## **Code:**

```coffeescript
# The following three Ractives don't actually render their components.  
new R
  template: ''' <C x=5/>  '''
  el: '#target3'
  
new R
  data:x:42
  template: ''' <C x={{x}}/>  '''
  el: '#target4'
  
new R
  data:x:42
  template: ''' <C x='{{x}}'/>  '''
  el: '#target5'
```

**Comment by ghost** [comment]
Confirming that it works on 0.7.3 https://codepen.io/anon/pen/WoGbZb

**Comment by trvrm** [comment]
In your version we get

`x is 5/`

i.e. ractive is interpreting the `/` as part of the variable being passed to the component, not as the end of the HTML tag.

**Comment by ghost** [comment]
I merely swapped out 0.8.5 with 0.7.3. Code is the same 😄 

However, it's been acting differently between versions:
- 0.4.0: https://codepen.io/anon/pen/vyXEpQ
- 0.5.8: https://codepen.io/anon/pen/XNjJZg
- 0.6.1: https://codepen.io/anon/pen/MbjYVJ
- 0.7.3: https://codepen.io/anon/pen/WoGbZb
- 0.8.5: https://codepen.io/anon/pen/LbREdr

I would defer to the HTML spec on this though to be consistent.

**Comment by evs-chris** [comment]
I've found the (simple) issue with `<C x={{x}}/>` in edge, and the fix is easy. These may be trickier to handle on 0.8, as edge has a good bit of refactoring that cleaned up attribute and directive parser, which is also why `<C x='{{x}}'/>` already didn't have an issue on edge.

**Comment by evs-chris** [comment]
Ah, the `<c x='{{x}}'/>` case _does_ work in 0.8, it just wasn't being run in the pen due to the error being thrown in the code above it.

**Comment by evs-chris** [comment]
Well, I was wrong, and the same simple fix also worked for 0.8 :smile: http://codepen.io/anon/pen/qqadWJ

This is now available in edge and v0.8-dev and will go out in 0.8.6. Thanks for the report!
